src/golang/lib/spark/utils.go

```diff
@@ -0,0 +1,137 @@
+package spark
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/aqueducthq/aqueduct/config"
+	"github.com/aqueducthq/aqueduct/lib"
+	exec_env "github.com/aqueducthq/aqueduct/lib/execution_environment"
+	"github.com/aqueducthq/aqueduct/lib/models/shared"
+	"github.com/aqueducthq/aqueduct/lib/storage"
+	"github.com/dropbox/godropbox/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+func CreateSparkEnv(
```

Comment:
* Maybe `CreateSparkEnvFile` or add some comments? We could be more explicit on this is creating a packed env in s3.
* Do we need `condaPath` as an input? I think this value is already baked in dkr img and it can be a constant, given even dkr imgs are constants.